### PR TITLE
verify Gradle

### PR DESCRIPTION
**Pull request by hannesa2**

### CODE REVIEW

**Comment by jesmrec** [comment]
it would be appreciated a descriptive title and a little sum up in the first message to know how this is about. Thanks for the contribution.

**Comment by hannesa2** [comment]
> it would be appreciated a descriptive title and a little sum up in the first message to know how this is about. Thanks for the contribution.

Here is the complete story https://github.com/marketplace/actions/gradle-wrapper-validation

**Comment by abelgardep** [comment]
Hi @hannesa2, could you update this pr?

**Comment by abelgardep** [comment]
Thanks for updating it.

After taking a deeper look at this gradle verifier, i think that checking it only at pull request would be enough. I consider that it is not needed to perform a gradle check on each commit. What do you think? Could you update it with this little change? Same for app workflow.

**Comment by hannesa2** [comment]
It runs on each push and this is fine

**Comment by abelgardep** [comment]
From my point of view, is not very common to modify the gradle and just a validation on the PR would be enough, as it is snyk and drone.

**Comment by hannesa2** [comment]
I don't know what you speaking about drone, this is my drone output

image

Please see it as inspiration and please feel free to improve it, the branch is on upstream repo

**Comment by jesmrec** [comment]
Approved

File: .github/workflows/gradle-wrapper-validation.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: "Validate Gradle Wrapper"
-on: [push, pull_request]
+on: [pull_request]
```
Comment (hannesa2) [inline .github/workflows/gradle-wrapper-validation.yml:2]: Now it's clear what you were meaning.
https://github.com/owncloud/android-library/pull/305#issuecomment-610949497

I did'm understood exactly what you were meaning

Comment (abelgardep) [reply .github/workflows/gradle-wrapper-validation.yml:2]: Ok, sorry, ill try to explain myself better next time

**Pull request merged**